src/test/java/org/apache/maven/shared/utils/cli/CommandLineUtilsTest.java

```diff
@@ -22,6 +22,7 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.hasItemInArray;
 import static org.hamcrest.Matchers.not;
+import static org.hamcrest.collection.IsArrayWithSize.emptyArray;
```

Comment:
This was unfamiliar and confusing when I read it below. A static import is not helpful here.